src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -243,7 +243,7 @@ public CompletableFuture<String> lookupAcquisitionUnitIdsByName(String data) {
   public CompletableFuture<Object> lookupAcquisitionUnitIdsByAccount(String data) {
 
     return lookupOrganization(ORGANIZATION_NAME).thenApply(org -> org.getAccounts().stream()
-      .filter(acc -> acc.getAccountNo().equals(HelperUtils.normalizeSubAccout(data)))
+      .filter(acc -> HelperUtils.normalizeSubAccout(acc.getAccountNo()).equals(HelperUtils.normalizeSubAccout(data)))
```

Comment:
This change needed because it was exists also in previos implementation:
![image](https://user-images.githubusercontent.com/25097693/188115368-462dd32a-0a76-4e7d-b67f-b4a00464f80d.png)

Reply:
Yes , And because GOBI organizations account no cantains "-".